recipe/meta.yaml

```diff
@@ -11,22 +11,24 @@ source:
   sha256: "{{ sha256 }}"
 
 build:
-  noarch: python
   number: 0
-  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vvv "
+  skip: True # [py<37]
+  script: "{{ PYTHON }} -m pip install . --no-deps --no-build-isolation --ignore-installed -vvv"
```

Comment:
Double quotes can be removed as the YAML key is already a string. The same applies to other places in the recipe.
The verbosity `-vvv` can be reduced to `-vv` for almost all Python packages